ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -380,6 +380,22 @@ The following `eos_cli_config_gen` inputs can be given as needed to generate the
 All changes except `"RANCID-CONTENT-TYPE: arista"` have been incorporated into `eos_designs`. Action is only required when using `eos_cli_config_gen` directly,
 or if the `RANCID-CONTENT-TYPE: arista` is required.
 
+### `router_isis.address_family_ipv4/6.enabled` is now `Required` to generate the configuration and documentation for router_isis address-families
+
+In AVD 4.0.0, the router-isis address-family configuration and documentation were generated independently of the `router_isis.address_family_ipv4/6.enabled` key.
+
+As of AVD 5.0.0, `router_isis.address_family_ipv4/6.enabled` must be set to generate the configuration and documentation for router_isis address-families.
+
+```diff
+  router_isis:
+    address_family_ipv6:
+-      enabled: <bool>
++      enabled: <bool; required>
+    address_family_ipv4:
+-       enabled: <bool>
++       enabled: <bool; required>
```

Comment:
We should not show the change to the data model here, but instead show what the customer should do to retain the same config.

so in this case:
```diff
 router_isis:
   address_family_ipv4:
+    enabled: true
     <...>
   address_family_ipv6:
     enabled: true
     <...>
```

Reply:
Fixed